test/ButtonSpec.js

```diff
@@ -5,7 +5,7 @@ import Button from '../src/Button';
 
 let wrapper = shallow(<Button waves='light'>Stuff</Button>);
 
-describe.only('Button', () => {
+describe('Button', () => {
```

Comment:
Thanks for pointing this out but this should live in another PR, just like other unrelated issues to the SideNav, checking the PR now!